docs/config/README.md

```diff
@@ -142,26 +154,36 @@ To use an HD Wallet with Hardhat you should set your network's `accounts` field
 }
 ```
 
-## Solc configuration
+## Solidity configuration
+
+The `solidity` config is an optional field that can be one of the following:
+
+- The solc version to use itself. E.g. `"0.7.3"`.
```

Comment:
```suggestion
- A solc version to use. E.g. `"0.7.3"`.
```

Reply:
I wonder if the default solidity version should be described as a default value here.

Reply:
There's a very annoying warning if you use the default value, so it doesn't matter that much.